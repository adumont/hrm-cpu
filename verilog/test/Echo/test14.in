24 f4
30 e7
10 6c
38 09
45 09
57 ee
30 41
44 5f
37 54
36 72
10 7c
10 2a
01 09
55 b3
31 d5
07 cb
01 cf
44 07
58 40
16 27
41 01
11 dc
16 98
38 09
43 2e
48 37
17 5e
53 5b
05 eb
45 e8
30 d8
54 f3
07 05
49 b5
26 a2
14 b0
19 d6
47 10
35 a9
25 bd
57 57
25 a1
37 57
05 90
20 26
22 cb
49 51
07 40
