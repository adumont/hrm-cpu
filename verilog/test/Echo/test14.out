f4
e7
6c
09
09
ee
41
5f
54
72
7c
2a
09
b3
d5
cb
cf
07
40
27
01
dc
98
09
2e
37
5e
5b
eb
e8
d8
f3
05
b5
a2
b0
d6
10
a9
bd
57
a1
57
90
26
cb
51
40
